pre-docs/inventory_version2.md

```diff
@@ -39,6 +39,43 @@ string to a target defined elsewhere in the inventory. If no target has a name
 or alias matching the string bolt will create a new target with the string as
 it's uri.
 
+### Migrating Plans
+
+In addition to inventory file changes, inventory functions such as `get_targets` may not work as expected when called from an [apply block](applying_manifest_blocks.md) using inventory version 2. `get_targets` will return an empty array when called with `'all'` as an argument, otherwise it will create a new Target object for that host. If you want to have the same behavior as the version 1 inventory, you can extract information outside of an apply block into a variable, and use that variable inside the apply block.
```

Comment:
Per Puppet style guide, recommend changing `may` to `might` since `may` can feel ambiguous to non-native English speakers (or even native speakers):
> ... might not work as expected when called from an ...

Reply:
Recommend present tense and breaking up run-on:
> `get_targets` returns an empty array when called with `'all'` as an argument. Otherwise, it creates a new Target object for that host.